File: app/api/join/join.py
# pip3 install requests
import requests
import json

API_KEY = "featlytalksfu_default_secret"
FEATLYTALK_URL = "https://talk.featly.io/api/v1/join"
# FEATLYTALK_URL = "http://localhost:3010/api/v1/join"

headers = {
    "authorization": API_KEY,
    "Content-Type": "application/json",
}

data = {
    "room": "test",
    "password": "false",
    "name": "featlytalksfu",
    "audio": "true",
    "video": "true",
    "screen": "true",
    "notify": "true",
}

response = requests.post(
    FEATLYTALK_URL,
    headers=headers,
    json=data,
)

print("Status code:", response.status_code)
data = json.loads(response.text)
print("join:", data["join"])

File: app/api/join/join.js
'use strict';

const fetch = require('node-fetch');

const API_KEY = 'featlytalksfu_default_secret';
// const FEATLYTALK_URL = 'https://featly.io/api/v1/join';
const FEATLYTALK_URL = 'https://localhost:3010/api/v1/join';

function getResponse() {
    return fetch(FEATLYTALK_URL, {
        method: 'POST',
        headers: {
            authorization: API_KEY,
            'Content-Type': 'application/json',
        },
        body: JSON.stringify({
            room: 'test',
            password: false,
            name: 'featlytalksfu',
            audio: true,
            video: true,
            screen: true,
            notify: true,
        }),
    });
}

getResponse().then(async (res) => {
    console.log('Status code:', res.status);
    const data = await res.json();
    console.log('join:', data.join);
});
